Silicon/NXP/iMX6Pkg/Library/iMX6ClkPwrLib/iMX6ClkPwr.c

```diff
@@ -1357,39 +1182,39 @@ ImxClkPwrValidateClocks (
   EFI_STATUS      Status;
 
   Invalid = FALSE;
-  for (i = 0; i < ARRAYSIZE (ExpectedClocks); ++i) {
+  for (i = 0; i < ARRAY_SIZE (mExpectedClocks); ++i) {
     DEBUG ((
             DEBUG_INFO,
-            "%a: Validating clock %s. Expecting: Frequency = %d (%d Mhz), Parent = %s\n",
+            "%a: Validating clock %d. Expecting: Frequency = %d (%d Mhz), Parent = %d\n",
             __FUNCTION__,
-            StringFromImxClk (ExpectedClocks[i].Clock),
-            ExpectedClocks[i].Info.Frequency,
-            ExpectedClocks[i].Info.Frequency / 1000000,
-            StringFromImxClk (ExpectedClocks[i].Info.Parent)
+            mExpectedClocks[i].Clock,
+            mExpectedClocks[i].Info.Frequency,
+            mExpectedClocks[i].Info.Frequency / 1000000,
+            mExpectedClocks[i].Info.Parent
           ));
-    Status = ImxClkPwrGetClockInfo (ExpectedClocks[i].Clock, &ActualInfo);
+    Status = ImxClkPwrGetClockInfo (mExpectedClocks[i].Clock, &ActualInfo);
     if (EFI_ERROR (Status)) {
       DEBUG ((
               DEBUG_ERROR,
-              "%a: Failed to get clock info. (Clock = %s, Status = 0x%x)\n",
+              "%a: Failed to get clock info. (Clock = %d, Status = 0x%x)\n",

```

Comment:
Any reason we're making the output less readable by changing from say SSI_CLK_ROOT to 14?

Reply:
I was looking to remove the StringFromImxClk function since it got negative attention in the upstream review for being unwieldy. This change was a byproduct of that removal.

I can add the string conversion function back in if we think it'll be useful to maintain going forward.